### PR TITLE
Sprint2 refactor back

### PR DESCRIPTION
**Pull request by BMB0**
Updated codebase by refactoring the commits controller and module for improved code clarity and maintainability

### CODE REVIEW

File: server/src/modules/Commits/Application/getCommits.ts [hunk 1]
```diff
@@ -3,19 +3,39 @@ import { CommitRepository } from "../Repositories/commitRepository";
 import { updateCommitsTable } from "./updateCommitsTable";
 
 
+// export const getCommits = async (
```
Comment (DwijanX) [inline server/src/modules/Commits/Application/getCommits.ts]: we should remove the commented code I think

Comment (DwijanX) [reply server/src/modules/Commits/Application/getCommits.ts]: nvm, it was deleted later xd

**Comment by DwijanX** [comment]
Everything looks fine, the only problem is that the deploy is failing @Ogreaggressive @reneddc

**Comment by DwijanX** [comment]
After merging @FranAliss  PR conficts appeared xd

**Comment by BMB0** [comment]
Resolved the conflicts after PR merge. it should be ready for merging

**Pull request merged**